pipeline/outputs/firehose.md

```diff
@@ -19,7 +19,7 @@ See [here](https://github.com/fluent/fluent-bit-docs/tree/43c4fe134611da471e706b
 | region | The AWS region. |
 | delivery\_stream | The name of the Kinesis Firehose Delivery stream that you want log records sent to. |
 | time\_key | Add the timestamp to the record under this key. By default the timestamp from Fluent Bit will not be added to records sent to Kinesis. |
-| time\_key\_format | strftime compliant format string for the timestamp; for example, the default is '%Y-%m-%dT%H:%M:%S'. This option is used with time\_key. |
+| time\_key\_format | strftime compliant format string for the timestamp; for example, the default is '%Y-%m-%dT%H:%M:%S'. Now can support millisecond with '%3N' and support nanosecond with '%9N' and '%L'; for example, adding '%3N' to support millisecond '%Y-%m-%dT%H:%M:%S:%3N'. This option is used with time\_key. |
```

Comment:
Docs are timeless, so do not use words like "now". Just "Supports millisecond with..."

Reply:
> '%Y-%m-%dT%H:%M:%S:%3N'

Should it be `.%3N` since that is what most users would configure?